.github/workflows/lint.yml

```diff
@@ -68,11 +68,15 @@ jobs:
         run: |
           echo "::add-matcher::.github/workflows/matchers/pylint.json"
           tox -e lint
+          echo "::remove-matcher owner=pylint-error::"
```

Comment:
We want mypy to run even when lint fails. But this only removes the pylint matcher when lint succeeds. We probably need to move the remove-matcher to a separate step that runs even when lint fails or into the mypy step.

Reply:
They do survive as proven here: https://github.com/instructlab/instructlab/pull/1286/files

Will update as you suggested.